llvm/utils/lldbDataFormatters.py

```diff
@@ -372,7 +378,8 @@ def update(self):
         # For each key, collect a list of buckets it appears in.
         key_buckets: dict[str, list[int]] = collections.defaultdict(list)
         for index in range(num_buckets):
-            key = buckets.GetValueForExpressionPath(f"[{index}].first")
+            bucket = buckets.GetValueForExpressionPath(f"[{index}]")
+            key = bucket.GetChildAtIndex(0)
```

Comment:
This change is because the entries of a DenseSet are `DenseSetPair`, which is not actually a pair. There's only one child, and its name is `key`. The original code works for `DenseMapPair`, which has two children, `first` and `second`. Using `GetChildAtIndex(0)` will retrieve `first` for maps, and `key` for sets.